webapp/packages/core-blocks/package.json

```diff
@@ -39,10 +39,13 @@
   },
   "peerDependencies": {},
   "devDependencies": {
+    "@cloudbeaver/core-di": "^0",
+    "@cloudbeaver/core-dialogs": "^0",
```

Comment:
a bit strange that we need `"@cloudbeaver/core-dialogs": "^0",` in core-blocks (because core-dialogs depends on core-blocks)